README.md

```diff
@@ -25,6 +25,37 @@ Examples are available in the [`yank-examples` repository](http://github.com/cho
 git clone https://github.com/choderalab/yank-examples.git
 ```
 
+Requirements
+------------
+
+**Operating System**
+* Linux - 64bit (Ubuntu 12.04+, RHEL 6+, etc) 
+* OS X 10.7 or later (10.10.3 or later for OpenCL)
+
+**Python Requirements (installed automatically through conda**
```

Comment:
Close parenthesis (funny how this feels terribly important after you learn programming)